schema/sqitch.plan

```diff
@@ -23,6 +23,7 @@ tables/cif_user 2021-10-21T16:10:32Z Matthieu Foucault <matthieu@button.is> # Cr
 trigger_functions/set_user_id 2021-10-21T16:10:32Z Matthieu Foucault <matthieu@button.is> # Create the cif_private.set_user_id trigger function
 mutations/create_user_from_session 2021-10-21T16:10:32Z Matthieu Foucault <matthieu@button.is> # Create the cif.create_user_from_session mutation function
 tables/funding_stream 2021-12-01T05:19:11Z Alex Zorkin <AizeeDev@BigThink> # adding funding_stream table to plan
+tables/funding_stream_rfp 2021-12-15T00:47:25Z Alex Zorkin <AizeeDev@BigThink> # add funding stream rfp table
```

Comment:
You might want to update the sqitch plan and your sqitch config to have an actual email address instead of `<AizeeDev@BigThink>`